### PR TITLE
[zh]update content/zh/docs/concepts/workloads/controllers/job.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -858,6 +864,61 @@ as soon as the Job was resumed.
...
+自定义队列控制器，以决定工作何时开始;然而，一旦工作不被暂停，
+自定义队列控制器对工作的 Pods 实际放置的位置没有影响。
+
+<!--
+This feature allows updating a Job's scheduling directives before it starts, which gives custom queue
+controllers the ability to influence pod placement while at the same time offloading actual 
+pod-to-node assignment to kube-scheduler. This is allowed only for suspended Jobs that have never 
+been unsuspended before.
+-->
+该特性允许在Job开始之前更新调度指令，从而提供定制队列
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/job.md:909]:
```suggestion
此特性允许在 Job 开始之前更新调度指令，从而为定制队列提供影响 Pod
放置的能力，同时将 Pod 与节点间的分配关系留给 kube-scheduler 决定。
这一特性仅适用于之前从未被暂停过的、已暂停的 Job。
```

File: content/zh/docs/concepts/workloads/controllers/job.md [hunk 2]
```diff
@@ -858,6 +864,61 @@ as soon as the Job was resumed.
...
+控制器能够影响 Pod 放置，同时参考实际
+pod-to-node 分配给 kube-scheduler。 这仅适用于从未暂停的Jobs。
+
+<!--
+The fields in a Job's pod template that can be updated are node affinity, node selector, 
+tolerations, labels and annotations.
+-->
+Job 的 Pod 模板中可以更新的字段是节点亲和性、节点选择器、
+容忍、标签和注释。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/job.md:918]:
```suggestion
Job 的 Pod 模板中可以更新的字段是节点亲和性、节点选择器、容忍、标签和注解。
```

File: content/zh/docs/concepts/workloads/controllers/job.md [hunk 3]
```diff
@@ -858,6 +864,62 @@ as soon as the Job was resumed.
...
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/).
+It is enabled by default.
+-->
+{{< note >}}
+为了使用此功能，你必须在[API 服务器](/zh/docs/reference/command-line-tools-reference/kube-apiserver/)上启用
+`JobMutableNodeSchedulingDirectives` [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)。
+。
+默认情况下启用。
+{{< /note >}}
```
Comment (Sea-n) [inline content/zh/docs/concepts/workloads/controllers/job.md:885]: Unknown dot (`。`) here.

```diff
 `JobMutableNodeSchedulingDirectives` [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)。
-。
 默认情况下启用。
```

File: content/zh/docs/concepts/workloads/controllers/job.md [hunk 4]
```diff
@@ -858,6 +864,62 @@ as soon as the Job was resumed.
...
+This feature allows updating a Job's scheduling directives before it starts, which gives custom queue
+controllers the ability to influence pod placement while at the same time offloading actual 
+pod-to-node assignment to kube-scheduler. This is allowed only for suspended Jobs that have never 
+been unsuspended before.
+-->
+此特性允许在 Job 开始之前更新调度指令，从而为定制队列提供影响 Pod
+放置的能力，同时将 Pod 与节点间的分配关系留给 kube-scheduler 决定。
+这一特性仅适用于之前从未被暂停过的、已暂停的 Job。
+控制器能够影响 Pod 放置，同时参考实际
+pod-to-node 分配给 kube-scheduler。 这仅适用于从未暂停的Jobs。
```
Comment (Sea-n) [inline content/zh/docs/concepts/workloads/controllers/job.md:913]: Here missing space between Chinese and English words.

```diff
 控制器能够影响 Pod 放置，同时参考实际
-pod-to-node 分配给 kube-scheduler。 这仅适用于从未暂停的Jobs。
+pod-to-node 分配给 kube-scheduler。这仅适用于从未暂停的 Jobs。
```

File: content/zh/docs/concepts/workloads/controllers/job.md [hunk 5]
```diff
@@ -858,6 +864,62 @@ as soon as the Job was resumed.
...
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+
+<!--
+In order to use this behavior, you must enable the `JobMutableNodeSchedulingDirectives`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/).
+It is enabled by default.
+-->
+{{< note >}}
+为了使用此功能，你必须在[API 服务器](/zh/docs/reference/command-line-tools-reference/kube-apiserver/)上启用
```
Comment (Sea-n) [inline content/zh/docs/concepts/workloads/controllers/job.md:881]: Missing space between Chinese and English word.
```diff
-你必须在[API 服务器]
+你必须在 [API 服务器]
```

**Review by Sea-n — CHANGES_REQUESTED** [review]
Please take a look at [排版格式](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%8E%92%E7%89%88%E6%A0%BC%E5%BC%8F).

File: content/zh/docs/concepts/workloads/controllers/job.md [hunk 6]
```diff
@@ -878,9 +878,9 @@ on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/
 It is enabled by default.
 -->
 {{< note >}}
-为了使用此功能，你必须启用 `JobMutableNodeSchedulingDirectives`
-[特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)
-在[API 服务器](/zh/docs/reference/command-line-tools-reference/kube-apiserver/)上。
+为了使用此功能，你必须在[API 服务器](/zh/docs/reference/command-line-tools-reference/kube-apiserver/)上启用
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/job.md]: 链接中首字母、尾字母是英文时，按中英文留白处理。
```suggestion
为了使用此功能，你必须在 [API 服务器](/zh/docs/reference/command-line-tools-reference/kube-apiserver/)上启用
```

Comment (kinzhi) [reply content/zh/docs/concepts/workloads/controllers/job.md]: This has been changed

**Pull request merged**